contracts/utils/contracts/src/RichErrors.sol

```diff
@@ -0,0 +1,53 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.5;
+
+import "./mixins/MRichErrors.sol";
+
+
+contract RichErrors is
+    MRichErrors
+{
+    // solhint-disable func-name-mixedcase
+    function StandardError(
```

Comment:
Can we add a natspec comment for this, given that it isn't used anywhere at the moment?